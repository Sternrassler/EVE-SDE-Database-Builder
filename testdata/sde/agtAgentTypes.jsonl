{"agentTypeID":1,"agentType":"Non-Agent","description":"Non-agent entity"}
{"agentTypeID":2,"agentType":"Basic Agent","description":"Basic mission agent"}
{"agentTypeID":7,"agentType":"Research Agent","description":"Research point generating agent"}
