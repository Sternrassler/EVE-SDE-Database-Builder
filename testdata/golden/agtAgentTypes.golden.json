[
  {
    "agentTypeID": 1,
    "agentType": "Non-Agent"
  },
  {
    "agentTypeID": 2,
    "agentType": "Basic Agent"
  },
  {
    "agentTypeID": 7,
    "agentType": "Research Agent"
  }
]